app/src/main/res/values/strings.xml

```diff
@@ -443,4 +444,13 @@
     <string name="post_votes">Post votes</string>
     <string name="comment_votes">Comment votes</string>
     <string name="moderation">Moderation</string>
+    <string name="registrations">Registrations</string>
```

Comment:
*Registrations* seems a bit long for the title, but I couldn't think of anything else. In the bottom bar I use *Apps* for short.